examples/knowledge_tuning/instructlab/document_collection/ibm-annual-report/qna.yaml

```
@@ -0,0 +1,100 @@
+
+    (1) Recast to reflect January 2024 segment changes.
+
+    Consulting signings grew 3.3 percent as reported (4.7 percent adjusted for currency) in 2024, driven by our strong performance in signings in the fourth quarter of 2024. This performance reflects our ability to address client demands and the contribution of our generative AI solutions that help clients accelerate their digital transformations. Our book-to-bill ratio over the trailing twelve months was 1.21. Book-to-bill represents the ratio of IBM Consulting signings to its revenue over the same period and is a useful indicator of the demand for our business over time.
+
+    Signings are management's initial estimate of the value of a client's commitment under a services contract within IBM Consulting. There are no third-party standards or requirements governing the calculation of signings. The calculation used by management involves estimates and judgments to gauge the extent of a client's commitment, including the type and duration of the agreement and the presence of termination charges or wind-down costs.
+
+    Contract extensions and increases in scope are treated as signings only to the extent of the incremental new value. Total signings can vary over time due to a variety of factors including, but not limited to, the timing of signing a small number of larger contracts. Signings associated with an acquisition will be recognized on a prospective basis.
+
+    Management believes the estimated values of signings disclosed provide an indication of our forward-looking revenue. Signings are used to monitor the performance of the business and viewed as useful information for management and shareholders. The conversion of signings into revenue may vary based on the types of services and solutions, contract duration, customer decisions, and other factors, which may include, but are not limited to, the macroeconomic environment."
+  questions_and_answers:
+    - question: What was the total value of IBM’s Consulting signings in 2024 and how did it compare to 2023?
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)